src/main/java/org/zendesk/client/v2/model/Field.java

```diff
@@ -261,7 +261,7 @@ public String toString() {
         return sb.toString();
     }
 
-    public static class Option {
+    public static class Option implements Serializable {
```

Comment:
Define serialVersionUID here or cached data may not be compatible between different versions of zendesk-java-client.